### PR TITLE
Feat/add chain streams support

### PR DESCRIPTION
**Pull request by aarmoa**
- Added support for chain streams and a new example module.

### CODE REVIEW

**Comment by aarmoa** [comment]
The code coverage in the PR decreases a little bit. But it will be increased in the PR we are preparing to add the low level components in the SDK that the TS SDK has

File: examples/chain_client/49_ChainStream.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+from google.protobuf import json_format
+
+from pyinjective.async_client import AsyncClient
+from pyinjective.composer import Composer
+from pyinjective.core.network import Network
+
+
+async def main() -> None:
+    network = Network.devnet()
```
Comment (achilleas-kal) [inline examples/chain_client/49_ChainStream.py:11]: The rest of the examples point to testnet, for consistency it's best we use testnet here too.

**Review by achilleas-kal — APPROVED** [review]

File: examples/chain_client/49_ChainStream.py (hunk 1 above)
Comment (aarmoa) [reply examples/chain_client/49_ChainStream.py:11]: Will change this

**Comment by aarmoa** [comment]
Changes are ready to be merged. I am just waiting for a confirmation on the chain stream endpoints for the different networks

**Pull request merged**